$NetBSD: patch-ab,v 1.3 2000/09/09 18:11:54 wiz Exp $

We don't want to find any installed pthread packages, but there is no
configure switch for this. -- wiz

--- configure.orig	Mon Oct 25 10:25:53 1999
+++ configure
@@ -1073,6 +1073,8 @@
 # It thinks the first close brace ends the variable substitution.
 test -z "$INSTALL_PROGRAM" && INSTALL_PROGRAM='${INSTALL}'
 
+test -z "$INSTALL_SCRIPT" && INSTALL_SCRIPT='${INSTALL}'
+
 test -z "$INSTALL_DATA" && INSTALL_DATA='${INSTALL} -m 644'
 
 
@@ -3984,6 +3986,7 @@
 
 
 
+if false; then
 echo $ac_n "checking for main in -lpthread""... $ac_c" 1>&6
 echo "configure:3989: checking for main in -lpthread" >&5
 ac_lib_var=`echo pthread'_'main | sed 'y%./+-%__p_%'`
@@ -4105,6 +4108,7 @@
   echo "$ac_t""no" 1>&6
 fi
 done
+fi
 
 
 echo $ac_n "checking for lchown""... $ac_c" 1>&6
@@ -4158,6 +4162,7 @@
   echo "$ac_t""no" 1>&6
 fi
 
+if false; then
 echo $ac_n "checking for pthread_attr_setstacksize""... $ac_c" 1>&6
 echo "configure:4163: checking for pthread_attr_setstacksize" >&5
 if eval "test \"`echo '$''{'ac_cv_func_pthread_attr_setstacksize'+set}'`\" = set"; then
@@ -4353,6 +4358,7 @@
 fi
 
 fi
+fi
 
 
 # Check whether --enable-DCE or --disable-DCE was given.
@@ -5011,6 +5017,7 @@
 s%@YACC@%$YACC%g
 s%@RANLIB@%$RANLIB%g
 s%@INSTALL_PROGRAM@%$INSTALL_PROGRAM%g
+s%@INSTALL_SCRIPT@%$INSTALL_SCRIPT%g
 s%@INSTALL_DATA@%$INSTALL_DATA%g
 s%@PERL@%$PERL%g
 s%@TEX@%$TEX%g
